pkg/services/ngalert/api/compat.go

```diff
@@ -61,6 +61,42 @@ func ProvisionedAlertRuleFromAlertRules(rules []*models.AlertRule) definitions.P
 	return result
 }
 
+// AlertQueriesFromApiAlertQuery converts a collection of definitions.AlertQuery to collection of models.AlertQuery
+func AlertQueriesFromApiAlertQuery(queries []definitions.AlertQuery) []models.AlertQuery {
```

Comment:
Is it possible to always use the same notation to name the functions? I see there are 2 ways of defining the function names `...From...` and `...To...`. It would be great if we only use one of them as we would always know, at a glance, what the function does (I personally prefer the `...To...` form but any would suffice if we only use one of them).

For example, I find it clear if I see 2 functions names:
- `ApiAlertQueryToAlertQueries` and `AlertQueriesToApiAlertQuery`
Instead of having:
- `AlertQueriesFromApiAlertQuery` and `AlertQueriesToApiAlertQueries`

Reply:
```suggestion
func AlertQueriesFromApiAlertQueries(queries []definitions.AlertQuery) []models.AlertQuery {
```

Reply:
Good point! I have renamed functions to have the same pattern